src/IW/App/Monad.hs

```diff
@@ -0,0 +1,48 @@
+module IW.App.Monad
+    ( -- * Application monad
+      App (..)
+    , AppEnv
```

Comment:
Reexports are not following the style guide here as well.

Reply:
Will fix!